rules/no-skip-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+module.exports = function (context) {
+	var ava = createAvaRule();
+	return ava.merge({
+		MemberExpression: function (node) {
+			if (ava.isTestFile && node.property.name === 'skip' && util.isCallFromTestObject(node)) {
```

Comment:
OK, that simplifies things. Saw a thread about integrating it, but I didn't know how Ava currently treated them (error, badly formatted message, crash...)